Documentation/policy/language.rst

```diff
@@ -524,20 +524,38 @@ Kafka (Tech Preview)
 
 PortRuleKafka is a list of Kafka protocol constraints. All fields are optional,
 if all fields are empty or missing, the rule will match all Kafka messages.
+There are two ways to specify the Kafka rules. We can choose to specify a
+high level "produce" or "consume" role to a topic or choose to specify more
+low level kafka protocol specific apiKeys. Thus the rule allows us to specify
+both high level role constructs as well as low level Kafka protocol constructs.
```

Comment:
high-level ... low-level

Reply:
These last two sentences appear to state the same thing. Perhaps replace the last sentence with something like:

"Writing rules based on Kafka roles is easier and covers most common use cases, however if more granularity is needed then users can alternatively write rules using specific apiKeys.

Reply:
sure.

Reply:
done